api/bonsai_api/routers/samples.py

```diff
@@ -107,7 +107,7 @@ async def samples_summary(
     skip: int = Query(0, gt=-1),
     prediction_result: bool = Query(True, description="Include prediction results"),
     qc_metrics: bool = Query(False, description="Include QC metrics"),
-    sid: list[str] = Query([], description="Optional limit query to samples ids"),
+    sid: list[str] | None = Query(None, description="Optional limit query to samples ids"),
```

Comment:
I think this change is fine. It is an interesting discussion hiding here, on whether the additional clarity of the optional `None` type is worth having to consider that the value might be both `[]` and `None`.

I think I have a slight preference for the previous syntax, avoiding the extra `None` check, but either is fine I guess.